data/tabular/train_test_split.py

```diff
@@ -0,0 +1,158 @@
+import tdc
+import pandas as pd
+import numpy as np
+from typing import List, Tuple, Dict, Union, Optional
+import sys
+from tqdm import tqdm
+from random import Random
+from collections import defaultdict
+from rdkit import Chem
+from rdkit.Chem.Scaffolds import MurckoScaffold
+from rdkit import RDLogger
+from glob import glob
+
+RDLogger.DisableLog("rdApp.*")
+
+
+REPRESENTATIONS = [
+    'SMILES',
+    'SELFIES',
+    'PSMILES',
+    'DeepSMILES',
+    'canonical SMILES',
+    'InChI',
+    'TUCAN',
+    'IUPAC name'
+    ]
```

Comment:
why do you pass them all into the function (?) if the function only takes SMILES?

Reply:
was thinking of maybe conversion tools between the different representations?